pkg/util/sys/device.go

```diff
@@ -92,8 +92,14 @@ func ListDevices(executor exec.Executor) ([]string, error) {
 }
 
 func GetDevicePartitions(device string, executor exec.Executor) (partitions []Partition, unusedSpace uint64, err error) {
-	cmd := fmt.Sprintf("lsblk /dev/%s", device)
-	output, err := executor.ExecuteCommandWithOutput(false, cmd, "lsblk", fmt.Sprintf("/dev/%s", device),
+	deviceName := device
+	devicePath := strings.Split(device, "/")
+	if len(devicePath) == 1 {
```

Comment:
What is this change for? Is this from some prototyping or is it still needed?

Reply:
As I mentioned earlier https://github.com/rook/rook/pull/3451#discussion_r311742119

Reply:
@travisn have you checked this?

Reply:
yes, makes sense thanks. Could you just add a comment here with that info?